mutation-testing/check-packages-and-shards/README.md

```diff
@@ -0,0 +1,28 @@
+# Check Packages and Shards action
+
+Checks whether to run mutants on big (`stackslib`/`stacks-node`) or small packages (all others), and whether to run them using strategy matrix or not.
```

Comment:
can you add a link to the stacks-core repo for the stackslib and stacks-node packages?